client/lib/js/react/components/asset_detail.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Row, Glyphicon } from 'react-bootstrap/lib';
+import classnames from 'classnames';
+
+
+const AssetDetail = React.createClass({
+    propTypes: {
+        asset: React.PropTypes.object,
+        assetContent: React.PropTypes.string,
+        children: React.PropTypes.oneOfType([
+            React.PropTypes.arrayOf(React.PropTypes.node),
+            React.PropTypes.node
+        ]),
+        className: React.PropTypes.string
+    },
+
+    getDefaultProps() {
+        return {
+            className: 'pull-right'
+        };
+    },
+
+    getAssetContent() {
+        const { asset, assetContent } = this.props;
+        if (!assetContent) {
+            return asset.transaction.data ? asset.transaction.data.payload.content : '-';
+        } else {
+            return assetContent;
+        }
+    },
+
+    render() {
+        const {
+            asset,
+            children,
+            className
+        } = this.props;
+
+        const assetContent = this.getAssetContent();
+        const validGlyph = asset.hasOwnProperty('assignee') ? <Glyphicon glyph="cog" />
```

Comment:
it's a way to see if the transaction came from the bigchain or backlog
I could give it a wrapper, but in the future I'd want to have a frontend model like:

```
assetList = {
    account: {
        bigchain: {
             [tx's]
        }
        backlog: {
             [tx's]
        }
    }
}
```